understanding/headings-and-labels.html

```diff
@@ -58,190 +56,133 @@
-                  and therefore pass Success Criterion 4.1.2, but to still fail this Success Criterion
-                  (if the label is not
-                  sufficiently clear or descriptive).
+               <p>さらに、ラベルの場合、この達成基準はフォームコントロール及び入力にアクセシブルな名前を提供する代替手段が用いられているか否かを考慮しない。この点は <a href="name-role-value">4.1.2: 名前 (name)・役割 (role)・値 (value)</a> で別途取り扱っている。コントロール及び入力に適したアクセシブルな名前 (例えば、<code>aria-label="…"</code> を使用して) を持ち、その結果達成基準 4.1.2 に合格したとしても、(ラベルが十分に明確又は説明的ではない場合) この達成基準に失敗する可能性がある。
                   
                </p>
-               <p>This success criterion does not require the use of labels; however, it does require
-                  that if labels are present, they must be sufficiently clear or descriptive. Please
-                  see <a href="labels-or-instructions">3.3.2: Labels or Instructions</a> for more information on the use of labels.
+               <p>この達成基準はラベルの使用を求めていない。しかし、ラベルが存在する場合、それらが十分に明確又は説明的でなければならないことは求めている。ラベルの使用の詳細については、<a href="labels-or-instructions">3.3.2: ラベル又は説明</a>を参照すること。
                   
                </p>
             </section>
             <section id="benefits">
-               <h2>Benefits</h2>
+               <h2>利点</h2>
                <ul>
                   
-                  <li>Descriptive headings are especially helpful for users who have disabilities that make
-                     reading slow and for people with limited short-term memory. These people benefit when
-                     section titles make it possible to predict what each section contains.
+                  <li>内容が分かるように記述された見出しは、読む速度が遅くなる障害のある利用者及び短期記憶に制約のある利用者に特に役立つ。そのような利用者にとっては、それぞれのセクションの内容を予測できるようにセクションの見出しが記述されていると役立つ。
                   </li>
                   
-                  <li>Form input controls with labels that clearly describe the content that is expected
-                     to be
-                     entered helps users know how to successfully complete the form.
+                  <li>入力が予想されるコンテンツを明確に説明するラベルの付いたフォーム入力コントロールは利用者がフォームに正しく入力するのを手助けする。
                   </li>
                   
-                  <li>When headings and labels are also correctly marked up and identified in accordance
-                     with
-                     <a href="info-and-relationships">1.3.1: Info and Relationships</a>, this Success Criterion
-                     helps people who use screen readers by ensuring that labels and headings are clearer
-                     when
-                     presented in a different format – for example, in an automatically generated list
-                     of
-                     headings, a table of contents, or when jumping from heading to heading within a page.
+                  <li>見出し及びラベルが <a href="info-and-relationships">1.3.1: 情報及び関係性</a>にも従って正しくマークアップ及び識別されている場合、ラベル及び見出しは異なるフォーマットで提示されても明快であることが保証されるので (例えば、自動的に生成される見出しリスト、目次、又はページ内の見出しから見出しへの移動)、この達成基準はスクリーンリーダーの利用者にとっても手助けとなる。
```

Comment:
語順を入れ替えてみましたがどうでしょうか。
```suggestion
                  <li>見出し及びラベルが <a href="info-and-relationships">1.3.1: 情報及び関係性</a>にも従って正しくマークアップ及び識別されている場合、この達成基準は、ラベル及び見出しは異なるフォーマット (例えば、自動的に生成される見出しのリスト、目次、又はページ内の見出しから見出しへの移動) で提示されても明快であることが保証されるので、スクリーンリーダーの利用者にとって手助けとなる。
```